src/test/java/Homework19.java

```diff
@@ -0,0 +1,48 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+import pages.PlaylistPage;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
```

Comment:
Please, **delete** `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

Reply:
Sorry I missed that. I deleted it! Thank you for the thorough review!

Reply:
> Sorry I missed that. I deleted it! Thank you for the thorough review!

I'm glad you found it helpful.